# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1502297440 25200
#      Wed Aug 09 09:50:40 2017 -0700
# Node ID 9f05f5afff45aa6839c151ae7b342c20b76d8790
# Parent  e43e95392c29131c4fa10bb314c8f043b0a73d72
maint: sort lists of files in test makefile snippets

* test/bug-35448/module.mk (bug_35448_TEST_FILES): Sort the list.
* test/bug-36025/module.mk (bug_36025_TEST_FILES): Likewise.
* test/bug-38236/module.mk (bug_38236_TEST_FILES): Likewise.
* test/bug-38691/module.mk (bug_38691_TEST_FILES): Likewise.
* test/bug-46660/module.mk (bug_46660_TEST_FILES): Likewise.
* test/bug-50035/module.mk (bug_50035_TEST_FILES): Likewise.
* test/bug-50716/module.mk (bug_50716_TEST_FILES): Likewise.
* test/bug-51532/module.mk (bug_51532_TEST_FILES): Likewise.
* test/bug-51534/module.mk (bug_51534_TEST_FILES): Likewise.
* test/classdef/module.mk (classdef_TEST_FILES): Likewise.
* test/fcn-handle-derived-resolution/module.mk
(fcn_handle_derived_resolution_TEST_FILES): Likewise.
* test/nest/module.mk (nest_TEST_FILES): Likewise.
* test/bug-50014/module.mk: Delete blank lines.
* test/module.mk (%reldir%/conv.tst, %reldir%/sparse.tst): Reorder rules.

diff --git a/test/bug-35448/module.mk b/test/bug-35448/module.mk
--- a/test/bug-35448/module.mk
+++ b/test/bug-35448/module.mk
@@ -1,7 +1,7 @@
 bug_35448_TEST_FILES = \
+  %reldir%/bug-35448.tst \
   %reldir%/fA.m \
   %reldir%/fB.m \
-  %reldir%/fC.m \
-  %reldir%/bug-35448.tst
+  %reldir%/fC.m
 
 TEST_FILES += $(bug_35448_TEST_FILES)
diff --git a/test/bug-36025/module.mk b/test/bug-36025/module.mk
--- a/test/bug-36025/module.mk
+++ b/test/bug-36025/module.mk
@@ -1,8 +1,7 @@
-
 bug_36025_TEST_FILES = \
   %reldir%/@testclass/one.m \
   %reldir%/@testclass/testclass.m \
   %reldir%/@testclass/two.m \
   %reldir%/bug-36025.tst
 
 TEST_FILES += $(bug_36025_TEST_FILES)
diff --git a/test/bug-38236/module.mk b/test/bug-38236/module.mk
--- a/test/bug-38236/module.mk
+++ b/test/bug-38236/module.mk
@@ -1,6 +1,6 @@
 bug_38236_TEST_FILES = \
+  %reldir%/bug-38236.tst \
   %reldir%/df_vr.m \
-  %reldir%/u_vr.m \
-  %reldir%/bug-38236.tst
+  %reldir%/u_vr.m
 
 TEST_FILES += $(bug_38236_TEST_FILES)
diff --git a/test/bug-38691/module.mk b/test/bug-38691/module.mk
--- a/test/bug-38691/module.mk
+++ b/test/bug-38691/module.mk
@@ -1,8 +1,8 @@
 bug_38691_TEST_FILES = \
+  %reldir%/bug-38691.tst \
   %reldir%/dir1/func1.m \
   %reldir%/dir2/func1.m \
   %reldir%/dir2/func2.m \
-  %reldir%/dir2/func3.m \
-  %reldir%/bug-38691.tst
+  %reldir%/dir2/func3.m
 
 TEST_FILES += $(bug_38691_TEST_FILES)
diff --git a/test/bug-46660/module.mk b/test/bug-46660/module.mk
--- a/test/bug-46660/module.mk
+++ b/test/bug-46660/module.mk
@@ -1,6 +1,5 @@
-
 bug_46660_TEST_FILES = \
-  %reldir%/class_bug46660.m \
-  %reldir%/bug-46660.tst
+  %reldir%/bug-46660.tst \
+  %reldir%/class_bug46660.m
 
 TEST_FILES += $(bug_46660_TEST_FILES)
diff --git a/test/bug-50014/module.mk b/test/bug-50014/module.mk
--- a/test/bug-50014/module.mk
+++ b/test/bug-50014/module.mk
@@ -1,9 +1,8 @@
-
 bug_50014_TEST_FILES = \
   %reldir%/bug-50014.tst \
   %reldir%/duplicate_nested_function.m \
   %reldir%/duplicate_nested_in_subfunction_ok.m \
   %reldir%/duplicate_nested_parent_function.m \
   %reldir%/duplicate_parent_nested2.m \
   %reldir%/duplicate_parent_nested_function.m \
   %reldir%/duplicate_primary_nested_function.m \
diff --git a/test/bug-50035/module.mk b/test/bug-50035/module.mk
--- a/test/bug-50035/module.mk
+++ b/test/bug-50035/module.mk
@@ -1,7 +1,6 @@
-
 bug_50035_TEST_FILES = \
+  %reldir%/@bug50035/bug50035.m \
   %reldir%/@bug50035/display.m \
-  %reldir%/@bug50035/bug50035.m \
   %reldir%/bug-50035.tst
 
 TEST_FILES += $(bug_50035_TEST_FILES)
diff --git a/test/bug-50716/module.mk b/test/bug-50716/module.mk
--- a/test/bug-50716/module.mk
+++ b/test/bug-50716/module.mk
@@ -1,6 +1,5 @@
-
 bug_50716_TEST_FILES = \
-  %reldir%/myclass.m \
-  %reldir%/bug-50716.tst
+  %reldir%/bug-50716.tst \
+  %reldir%/myclass.m
 
 TEST_FILES += $(bug_50716_TEST_FILES)
diff --git a/test/bug-51532/module.mk b/test/bug-51532/module.mk
--- a/test/bug-51532/module.mk
+++ b/test/bug-51532/module.mk
@@ -1,5 +1,5 @@
 bug_51532_TEST_FILES = \
-  %reldir%/bug-51532.tst \
-  %reldir%/+package_bug51532/foo.m
+  %reldir%/+package_bug51532/foo.m \
+  %reldir%/bug-51532.tst
 
 TEST_FILES += $(bug_51532_TEST_FILES)
diff --git a/test/bug-51534/module.mk b/test/bug-51534/module.mk
--- a/test/bug-51534/module.mk
+++ b/test/bug-51534/module.mk
@@ -1,6 +1,6 @@
 bug_51534_TEST_FILES = \
   %reldir%/bug-51534.tst \
-  %reldir%/parent_bug51534.m \
-  %reldir%/child_bug51534.m
+  %reldir%/child_bug51534.m \
+  %reldir%/parent_bug51534.m
 
 TEST_FILES += $(bug_51534_TEST_FILES)
diff --git a/test/classdef/module.mk b/test/classdef/module.mk
--- a/test/classdef/module.mk
+++ b/test/classdef/module.mk
@@ -1,10 +1,10 @@
 classdef_TEST_FILES = \
+  %reldir%/classdef.tst \
   %reldir%/foo_method_changes_property_size.m \
   %reldir%/foo_static_method_constant_property.m \
   %reldir%/foo_value_class.m \
-  %reldir%/classdef.tst \
   %reldir%/plist_t1.m \
   %reldir%/plist_t2.m \
   %reldir%/plist_t3.m
 
 TEST_FILES += $(classdef_TEST_FILES)
diff --git a/test/fcn-handle-derived-resolution/module.mk b/test/fcn-handle-derived-resolution/module.mk
--- a/test/fcn-handle-derived-resolution/module.mk
+++ b/test/fcn-handle-derived-resolution/module.mk
@@ -1,11 +1,11 @@
 fcn_handle_derived_resolution_TEST_FILES = \
   %reldir%/@fhdr_derived/fhdr_derived.m \
+  %reldir%/@fhdr_other/fhdr_other.m \
   %reldir%/@fhdr_other/getsize_arrayfun.m \
   %reldir%/@fhdr_other/getsize_cellfun.m \
   %reldir%/@fhdr_other/getsize_loop.m \
-  %reldir%/@fhdr_other/fhdr_other.m \
+  %reldir%/@fhdr_parent/fhdr_parent.m \
   %reldir%/@fhdr_parent/numel.m \
-  %reldir%/@fhdr_parent/fhdr_parent.m \
   %reldir%/fcn-handle-derived-resolution.tst
 
 TEST_FILES += $(fcn_handle_derived_resolution_TEST_FILES)
diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -115,26 +115,26 @@ coverage: all
 	@echo "Code coverage report successfully built.  Open the file"
 	@echo ""
 	@echo "   $(abs_top_builddir)/$(COVERAGE_DIR)/index.html"
 	@echo ""
 	@echo "in a web browser to view the results."
 	@echo ""
 .PHONY: coverage
 
+%reldir%/conv.tst: %reldir%/mk-conv-tst.sh | %reldir%/$(octave_dirstamp)
+	$(AM_V_GEN)rm -f $@-t $@ && \
+	$(SHELL) $(srcdir)/%reldir%/mk-conv-tst.sh > $@-t && \
+	mv $@-t $@
+
 %reldir%/sparse.tst: %reldir%/mk-sparse-tst.sh | %reldir%/$(octave_dirstamp)
 	$(AM_V_GEN)rm -f $@-t $@ && \
 	$(SHELL) $(srcdir)/%reldir%/mk-sparse-tst.sh > $@-t && \
 	mv $@-t $@
 
-%reldir%/conv.tst: %reldir%/mk-conv-tst.sh | %reldir%/$(octave_dirstamp)
-	$(AM_V_GEN)rm -f $@-t $@ && \
-	$(SHELL) $(srcdir)/%reldir%/mk-conv-tst.sh > $@-t && \
-	mv $@-t $@
-
 GENERATED_BC_OVERLOADS_DIRS := \
   $(shell $(SHELL) $(srcdir)/%reldir%/mk-bc-overloads-tst.sh test --list-dirs)
 
 GENERATED_BC_OVERLOADS_FILES := \
   $(shell $(SHELL) $(srcdir)/%reldir%/mk-bc-overloads-tst.sh test --list-files)
 
 $(GENERATED_BC_OVERLOADS_FILES): %reldir%/mk-bc-overloads-tst-stamp
 
diff --git a/test/nest/module.mk b/test/nest/module.mk
--- a/test/nest/module.mk
+++ b/test/nest/module.mk
@@ -1,20 +1,20 @@
 nest_TEST_FILES = \
   %reldir%/arg_nest.m \
   %reldir%/arg_ret.m \
+  %reldir%/nest.tst \
   %reldir%/nest_eval.m \
   %reldir%/no_closure.m \
   %reldir%/persistent_nest.m \
   %reldir%/recursive_nest.m \
   %reldir%/recursive_nest2.m \
   %reldir%/recursive_nest3.m \
   %reldir%/scope0.m \
   %reldir%/scope1.m \
   %reldir%/scope2.m \
   %reldir%/scope3.m \
   %reldir%/script_nest.m \
   %reldir%/script_nest_script.m \
-  %reldir%/nest.tst \
   %reldir%/varg_nest.m \
   %reldir%/varg_nest2.m
 
 TEST_FILES += $(nest_TEST_FILES)
